frontend/lib/evictionfree/declaration-builder/covid-impact.tsx

```diff
@@ -1,81 +1,91 @@
-        leave, that you have received since the start of the COVID-19 pandemic
-        does not fully make up for your loss of household income or increased
-        expenses.
-      </li>
+      <Trans id="evictionfree.financialHardshipExplainer2">
+        <li>
+          Significant loss of household income during the COVID-19 pandemic.
+        </li>
+        <li>
+          Increase in necessary out-of-pocket expenses related to performing
+          essential work or related to health impacts during the COVID-19
+          pandemic.
+        </li>
+        <li>
+          Childcare responsibilities or responsibilities to care for an elderly,
+          disabled, or sick family member during the COVID-19 pandemic have
+          negatively affected your ability or the ability of someone in your
+          household to obtain meaningful employment or earn income or increased
+          your necessary out-of-pocket expenses.
+        </li>
+        <li>
+          Moving expenses and difficulty you have securing alternative housing
+          make it a hardship for you to relocate to another residence during the
+          COVID-19 pandemic.
+        </li>
+        <li>
+          Other circumstances related to the COVID-19 pandemic have negatively
+          affected your ability to obtain meaningful employment or earn income
+          or have significantly reduced your household income or significantly
+          increased your expenses.
+        </li>
+        <li>
+          To the extent that you have lost household income or had increased
+          expenses, any public assistance, including unemployment insurance,
+          pandemic unemployment assistance, disability insurance, or paid family
+          leave, that you have received since the start of the COVID-19 pandemic
+          does not fully make up for your loss of household income or increased
+          expenses.
+        </li>
+      </Trans>
     </ul>
   </Accordion>
 );
 
 const HealthRiskAccordion: React.FC<{}> = () => (
-  <Accordion question="What does “significant health risk” mean?">
+  <Accordion question={li18n._(t`What does “significant health risk” mean?`)}>
     <p>
-      This means you or one or more members of your household have an increased
-      risk for severe illness or death from COVID-19 due to being over the age
-      of sixty-five, having a disability or having an underlying medical
-      condition, which may include but is not limited to being
-      immunocompromised.
+      <Trans id="evictionfree.significantHealthRiskExplainer">
+        This means you or one or more members of your household have an
+        increased risk for severe illness or death from COVID-19 due to being
+        over the age of sixty-five, having a disability or having an underlying
+        medical condition, which may include but is not limited to being
+        immunocompromised.
+      </Trans>
```

Comment:
As with the other question's answer, this was taken from the declaration but then converted to second-person.